### ISSUE TITLE
Get vertical flying machine

### ISSUE THREAD

**Issue by schrum2** [schrum2/MM-NEAT#811]
It would be nice to have one example of a vertical flying machine. Take the specific batch file that has been most successful in making flying machines (vector to volume, north south, accumulate), and make a new batch file that copies this one, but replaces north/south with up/down.

This might require some changes to the code as well, such as a new command line parameter.

**Comment by alejmedinajr** [comment]
The latest commit has what I think should work, except I cannot fully test it since I am on my personal laptop at the moment. I think I am missing something on my personal machine because tests that were working regularly and that we run in the labs do not work on here. After verifying if this works tomorrow, I will run tests to hopefully generate a vertical flying machine.

**Comment by alejmedinajr** [comment]
Update, I fixed the issue with my personal laptop and the vertical orientation restrictions seem to be working. I will test it tomorrow.

**Comment by MuellMark** [comment]
We have found out that observers can not be spawned in oriented up or down (North south, east and west, however, all work). When the orientation for up or down is selected, the observer spawns in facing north. I've run some tests trying to manually create a flying machine with this restriction, and I have verified that it is possible. The image below (and link) showcase this off:

![image](https://user-images.githubusercontent.com/92736704/176009574-eca738f7-f563-4c47-bdf0-97a9df7cefc8.png)
https://www.reddit.com/r/redstone/comments/i717y2/finally_figured_out_how_to_build_a_flying_machine/

Obviously we'd have more opportunities to generate a vertical flying machine with the up and down orientations, but it's good to see that it is still possible to generate a vertical flying machine without them
